go-controller/pkg/ovn/base_network_controller_multicast.go

```diff
@@ -107,8 +107,15 @@ func getNamespaceMcastACLDbIDs(ns string, aclDir libovsdbutil.ACLDirection, cont
 		})
 }
 
+func getNamespaceMcastPortGroupDbIDs(ns string, controller string) *libovsdbops.DbObjectIDs {
```

Comment:
I think moving `getNamespaceMcastPortGroupDbIDs`, and `getDefaultDenyPolicyPortGroupIDs` and other get DBIDs into libovsdb.utils pkg will be nice to do in future, but for now this makes sense to keep them all in base controller pkg since everything seems to be based off of that.